mne/tests/test_cov.py

```diff
@@ -588,7 +588,7 @@ def test_compute_covariance_auto_reg():
                         cov_b['data'][diag_mask]))
 
             # but the rest is the same
-            assert_array_equal(
+            assert_array_almost_equal(  # XXX make sure this is ok.
```

Comment:
What does this comment mean?

Reply:
That assert_array_equal does not work here and that I don't know if this is expected or not.

Reply:
I'd use `assert_allclose` with proper `rtol` and `atol` set based on the data of interest